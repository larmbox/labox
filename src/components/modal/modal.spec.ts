import { mount } from '@vue/test-utils';
import { beforeAll, expect, test } from 'vitest';
import { createLabox, LModal } from '../../';

beforeAll(() => {
  createLabox();
});

test('mount component', async () => {
  expect(LModal).toBeTruthy();

  const wrapper = mount(LModal, {
    props: {
      id: 'modal',
    },
  });

  expect(wrapper.html()).toMatchSnapshot();
  expect(wrapper.html()).toContain('modal');
});
