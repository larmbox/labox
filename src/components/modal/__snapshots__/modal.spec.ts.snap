// Vitest Snapshot v1

exports[`mount component 1`] = `
"<div id=\\"lx-modal-ref\\" style=\\"display: inline;\\">
  <!--v-if-->
</div>"
`;
